### ISSUE TITLE
Team colours swapped on panels?

### ISSUE THREAD

**Issue by pjmagee** [Ahli/Galaxy-Observer-UI#153]
![fClOXcr](https://user-images.githubusercontent.com/292720/103182560-1f2fe100-48a4-11eb-966c-32a5303861b2.jpg)

**Comment by Ahli** [comment]
Is that repeatable? Does it happen every time?
What needs to happen to break the bottom panel?
Can you send me the replay?

I have ideas how that could be fixed. But I would require a reliable way to test this.

**Comment by pjmagee** [comment]
Yeah, im not sure. It was happening on the twitch channel and I noticed it and took a quick screen grab so not to forget.

**Comment by pjmagee** [comment]
Okay! I found another occurrence when trying to find the original one. Here is the link to where the replay will be loaded. https://www.twitch.tv/videos/850626484?t=46h7m1s

**Comment by pjmagee** [comment]
I dont know which replay it is, but i have a large collection of them I can send you...not sure how helpful. you'd have to find out which one it is because im not sure... :( HOWEVER, it seems to be when values are all currently still set to 0 and when the game starts and panels cycle.

**Comment by Ahli** [comment]
The slots restored themselves later in that match when Muradin finishes his quest: https://www.twitch.tv/videos/850626484?t=46h19m42s
The camera is focused on Cassia during that. She is using Fend as seen in the progress bar.

Is the bottom panel opened before the heroes are created? There were issues in the past with it. Also the player slots could be weird. There were issues with opening the bottom panel in player vision many years ago, too. But I have not seen an issue like this in the past few years... =/

**Comment by Ahli** [comment]
I was able to recreate it via spamming red team's player slot number at the map start and then opening the bottom panel. But that does not explain how it changed in the videos
